styles/less/rules_index.less

```diff
@@ -10,3 +10,11 @@
         margin: 0;
     }
 }
+
+.replaced-by {
+    a + a:before {
+        content: ",";
+        display: inline-block;
```

Comment:
@ilyavolodin small caveat: had to use `inline-block` instead of `inline`, otherwise the comma would get the hover effect of the link like this:
<img width="363" alt="screenshot 2016-09-14 10 41 05" src="https://cloud.githubusercontent.com/assets/626038/18505941/e98ead9a-7a69-11e6-8284-87eae1c15e0a.png">